src/tree/box.js

```diff
@@ -139,6 +139,14 @@ export default function box(
       tooltip
         .html('')
         .attr('style', 'visibility: hidden;opacity: 0;');
+    })
+    .on('wheel', () => {
+      clearTimeout(tooltipOpen);
+      tooltipOpen = -1;
+      clearTimeout(tooltipClose);
+      tooltip
+        .html('')
+        .attr('style', 'visibility: hidden;opacity: 0;');
```

Comment:
Should consider moving the tooltip html def into its own file, basically allowing you to call tooltip.open(data) and tooltip.close().

You could pass in the tooltip element to those functions, making them akin to stateless.
Also, you can avoid setting .html('') each time, it should not be needed?

Reply:
yeah, that's a good idea 👍